scripts/actions/tool.actions.js

```diff
@@ -44,6 +44,65 @@ export function resetState(refilter = true) {
     dispatch(loadLinks());
   };
 }
+
+// Resets sankey's params that may lead to no flows being returned from the API
+export function resetSankey() {
+  return (dispatch, getState) => {
+    const state = getState();
+    const context = state.tool.contexts.find(context => context.id === state.tool.selectedContextId);
+    const defaultColumns = state.tool.columns.filter(column => column.isDefault);
+    const defaultResizeBy = context.resizeBy.find(resizeBy => resizeBy.isDefault);
```

Comment:
this is an unsafe access. I would add a fallback `context && context.resizeBy`

Reply:
true, but handling a case where a context is missing would be a totally different beast, so I'm adding that validation, but not really handling the case where context is missing